admin_pages/venues/templates/event_venues_metabox_content_from_manager.template.php

```diff
@@ -28,7 +28,7 @@
                 $state_name   = is_object($venue->state_obj()) ? $venue->state_obj()->name() : null;
                 $country_name = is_object($venue->country_obj()) ? $venue->country_obj()->name() : null;
                 ?>
-                <div class='eebox <?php echo $selected; ?>' id="eebox_<?php echo esc_attr($venue->ID()); ?>">
+                <div class='eebox <?php echo sanitize_html_class($selected); ?>' id="eebox_<?php echo esc_attr($venue->ID()); ?>">
```

Comment:
man this whole situation makes me upset (not what you are doing, i'm mad at wp.org about this, you're doing great)

`$selected` is a hard coded value that is set on line 23 above so imho, it does not require escaping, but they have pointed out situations exactly like this as requiring sanitization.

anyways... I know you didn't do this as part of this PR, but can you plz change

```php
echo esc_attr($venue->ID());
```

to:

```php
echo absint($venue->ID());
```

Reply:
I know it's somehow ridiculous to escape a variable that you defined yourself and impossible to change. But as far as I know, those guys at wp.org do a search on whole project to find anything matched with `echo $` and if they found anything, they will ask us to fix without checking the logic of code 🤦‍♂️. I already experienced it on other projects.

Reply:
ya that's exactly what they do, it's a completely brainless approach that just degrades the quality of code imho.